_docs/developer/troubleshooting/installation_troubleshooting.md

```diff
@@ -44,6 +44,17 @@ with no explanation, then there are a couple of things that may be going wrong:
    This will delete all virtual machine settings. Then install
    the latest version of Virtual Box and vagrant from the links given in step 3 (using Ubuntu Software).
 
+* If you are not using the prebuilt Submitty VM, and you get an error like
+  ```
+  FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmpad2cfj3v/abernl'
+  ```
+  This is a known issue with our implementation of Git gradeables, which will be resolved when we have the time. Until then, you can do.
+  ```
+  NO_SUBMISSIONS=1 vagrant up
```

Comment:
So, instead of `NO_SUBMISSIONS=1`, I have been using your branch `minus_git` as I often need to work with gradeables. I know Barb has the ultimate fix ready soon, but until then should we rather encourage devs to vagrant up from that branch instead? (and update doc)

Reply:
Since Barb has fixed Git gradeables, I have removed the tip from the docs.